Base/usr/share/man/man1/tail.md

```diff
@@ -4,13 +4,13 @@ tail
 
 ## Synopsis
 
-```sh
-$ tail [--follow] [--lines number] <file>
+```*sh
+$ tail [--follow] [--lines number] [file]
 ```
 
 ## Description
 
-Print the end ('tail') of a file.
+Print the end ('tail') of a file. If no file is specified, or the filename '-' is specified, it defaults to standard input.
```

Comment:
This should just be squashed into the previous commit in my opinion.

Reply:
I would personally disagree. Adding functionality and updating documentation are two separate tasks and so in my mind should be in two separate commits.

Reply:
You already updated the help message (and the utility itself) in the first commit, so the manpage should stay consistent. If you were to upload only the first commit, the linter would also tell you that the manpage is not up to date.

Reply:
Fair enough. I've squashed the commits into one now.